packages/xod-arduino/src/Directives.re

```diff
@@ -127,6 +127,13 @@ let doesRaiseErrors = (code) =>
   |. Code.lastPragmaEndis("error_raise")
   |. Endis.toBoolean(Code.doesReferSymbol("raiseError", code));
 
+let implementsEvaluateTmpl = Code.doesReferSymbol("evaluateTmpl");
```

Comment:
...or `#pragma`. `#pragma` should always be checked. Searching for a symbol is an unstable way. Say, I can *refer* to a symbol `evaluateTmpl` and it would be wrongly considered as the definition.

Reply:
Done, the pragma is named `evaluate_tmpl`